packages/passport/sdk-sample-app/src/context/ImmutableProvider.tsx

```diff
@@ -25,18 +25,30 @@ import { ImxApiClients, createConfig } from '@imtbl/generated-clients';
 const getSdkConfig = (environment: EnvironmentNames) => {
   switch (environment) {
     case EnvironmentNames.PRODUCTION: {
-      return production({});
+      const baseConfig = new ImmutableConfiguration({ environment: Environment.PRODUCTION });
+      return {
+        baseConfig,
+        override: production({}),
+      };
     }
     case EnvironmentNames.SANDBOX: {
-      return sandbox({});
+      const baseConfig = new ImmutableConfiguration({ environment: Environment.SANDBOX });
+      return {
+        baseConfig,
+        override: sandbox({}),
+      };
     }
     case EnvironmentNames.DEV: {
-      return createImmutableXConfiguration({
-        basePath: 'https://api.dev.x.immutable.com',
-        chainID: 5,
-        coreContractAddress: '0xd05323731807A35599BF9798a1DE15e89d6D6eF1',
-        registrationContractAddress: '0x7EB840223a3b1E0e8D54bF8A6cd83df5AFfC88B2',
-      });
+      const baseConfig = new ImmutableConfiguration({ environment: Environment.SANDBOX });
```

Comment:
does this look right? DEV's case has baseconfig about Sandbox environment.

Reply:
yeah...the way it works is that sandbox is used as the base, but overriden with details from dev.